server/db/create_test_data.js

```diff
@@ -116,9 +128,4 @@ program
   .parse(process.argv);
 
 var count = program.count || defaultCount;
-truncateTables(function(err) {
-  if (err) {
-    throw err;
-  }
-  createTestUser.call(null, count, done);
-});
+createTestUser(count, done);
```

Comment:
This could probably just be `createTestUser(program.count)` since the `createTestUser()` func already defaults to a `defaultCount` (see line 43).